front/src/modules/companies/components/CompanyChip.tsx

```diff
@@ -31,6 +32,12 @@ const StyledContainer = styled.span`
   }
 `;
 
+const StyledName = styled.span`
```

Comment:
I think it would be great to have storybook tests to avoid regressions! Could you add some (either at Chip level) or include long names in mock data for Pages storybook testing